docs/source/standard.md

```diff
@@ -39,67 +39,92 @@ What the TRE organisation does to ensure information risk is measured and manage
 
 ### 1.1 Compliance, monitoring and reporting
 
-The ability of the TRE organisation to monitor compliance with internal and external requirements, agreements, laws and standards.
+*The ability of the TRE organisation to monitor compliance with internal and external requirements, agreements, laws and standards.*
 
-| Statement | Guidance |
-| --------- | -------- |
-|           |          |
+| Statement | Guidance | Mandatory status |
+| --------- | -------  | --- |
+| You are able to audit your TRE organisation against relevant requirements and standards | If you are publicly accredited against a standard, for instance ISO27001, DSPT, CE+ or the SATRE specification, you must have processes in place to ensure you remain compliant | Mandatory |
+| You report on and share outcomes of each audit of your TRE organisation with the required bodies | This may included regulatory bodies or the organisations that manage accreditations you have | Mandatory |
 
 ### 1.2 Policy regulation and management
 
-TODO: someone needs to write a sentence
+*How an organsation determines what policies and regulations are required and ensures alignment to changes in requirements.*
 
-| Statement | Guidance |
-| --------- | -------- |
-|           |          |
+| Statement | Guidance | Manditory Status
+| --------- | -------- | ----------- |
+| You have a process in place to ensure any new project requiring a TRE meets relevant legal, ethical and contractual requirements | For example national legislation such as GDPR, discipline specific regulation like GCP or contractural requirements from a specific data provider such as a company or research partner organisation | Mandatory
+| You have a process in place to monitor changes to any legal, ethical and contractual requirements, and to update your policies accordingly | | Mandatory |
 
 ### 1.3 Quality management
 
-The ability of the TRE organisation to measure and control quality of processes, documentation and outputs.
+*The ability of the TRE organisation to measure and control quality of processes, documentation and outputs.*
 
-| Statement | Guidance |
-| --------- | -------- |
-|           |          |
+#### Document management
 
-#### 1.4 Audit and reporting
+| Statement | Guidance | Mandatory status |
+| --------- | -------- | ---------------- |
+| All policies & standard operating procedures relevant to the TRE organisation are controlled | This may include measures like restricting edit access to relevant documents, and recording acceptance of policies for all TRE organisation members | Mandatory |
```

Comment:
Do we also need some requirement to have any of the following?:
- up-to-date
- well-understood by users/operators
- actually followed
- version controlled
- codified change process

Reply:
I agree explicitly for version control and codified change process, will add